### DISCUSSION TITLE
Missing icons?

### DISCUSSION THREAD

**Discussion by hoclun-rigsep** [day8/re-com#321] [Q&A]
Hello. Beginner to Clojure(Script) and re-com. Love the library. I rsynced the static resources into my project but I seem to be missing some but not all of the icons. Is this expected?

**Comment by superstructor** [comment]
@hoclun-rigsep Thanks 😄 Hope you stay in Clojure(Script) a long time. Are you including the material design iconic font as per this example - https://github.com/day8/re-com/blob/master/run/resources/public/index_dev.html#L8 ?

**Comment by hoclun-rigsep** [comment]
Follow-up: I discovered that I had to write `zmdi-close-circle-o` rather than `close-circle-o`

```clojure
[row-button :src (at)
       :md-icon-name    "close-circle-o"
       ; :mouse-over-row? mouse-over-row?
       :tooltip         "Deny"
       ; :on-click        #(reset! click-msg (str "edit row " (:id row)))
       ]
```
